src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -31,7 +31,7 @@ public class ResolvePublishAssemblies : TaskBase
 
         public string PlatformLibraryName { get; set; }
 
-        public ITaskItem[] PrivateAssetsPackageReferences { get; set; }
+        public ITaskItem[] ExcludeFromPublishPackageReferences { get; set; }
```

Comment:
I think semver is OK here (along similar lines that convinced me for Cli.Utils + the fact that there is no ship vehicle for consuming as a library). I believe there have been other breaks.